code/modules/mob/living/carbon/human/species/monkey.dm

```diff
@@ -53,8 +54,9 @@
 	..()
 
 /datum/species/monkey/handle_dna(var/mob/living/carbon/human/H)
-	H.dna.SetSEState(MONKEYBLOCK,1)
-	genemutcheck(H, MONKEYBLOCK)
+	if(!is_type_in_list(/datum/dna/gene/monkey, H.active_genes)) //Only activate the monkey gene if it isn't already.
+		H.dna.SetSEState(MONKEYBLOCK,1,1)
+		genemutcheck(H,MONKEYBLOCK,null,MUTCHK_FORCED)
```

Comment:
This seems like another one of those things that probably doesn't need to be changed...